### PR TITLE
zh: what's new in v1.1

### LINKED ISSUE #315: What's New in 1.1

### PR DESCRIPTION
**Pull request by cofyc**
### What is changed, added or deleted? (Required)
fixes #315 

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/whats-new-in-v1.1.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: What's New in v1.1
```
Comment (DanielZhangQD) [inline zh/whats-new-in-v1.1.md:2]: Please update according to the comments on the google doc.

Comment (cofyc) [reply zh/whats-new-in-v1.1.md:2]: updated

File: zh/whats-new-in-v1.1.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+- 新增 `TidbMonitor` 用于部署集群监控
+- 新增 `TidbInitializer` 用于初始化集群
+- 新增 `Backup`、`BackupSchedule`、`Restore` 用于备份恢复集群
+- 新增 `TidbClusterAutoScaler` 实现[集群自动伸缩功能](enable-tidb-cluster-auto-scaling.md) (开启 `AutoScaling` 特性开关后使用)
+
+## 其他
+
+- 新增可选的[准入控制器](enable-admission-webhook.md)改进升级、扩缩容体验，并提供灰度发布功能
+- 新增可选的[增强型 StatefulSet 控制器](advanced-statefulset.md)，提供对指定 Pod 进行删除的功能 (开启 `AdvancedStatefulSet` 特性开关后使用)
```
Comment (ran-huang) [inline zh/whats-new-in-v1.1.md:28]:
```suggestion
- 新增可选的[增强型 StatefulSet 控制器](advanced-statefulset.md)，提供对指定 Pod 进行删除的功能（开启 `AdvancedStatefulSet` 特性开关后使用）
```

File: zh/whats-new-in-v1.1.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+- `tidb-scheduler` 支持任意维度的 HA 调度和 Preemption 支持
+- 备份、恢复支持 S3 和 GCS
```
Comment (ran-huang) [inline zh/whats-new-in-v1.1.md:30]:
```suggestion
- 备份与恢复支持将 S3 和 GCS 作为远端存储
```

File: zh/whats-new-in-v1.1.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+- 新增 Pump、TiFlash、TiCDC 支持
+- 新增 TiDB Dashboard  支持
+- 全面支持 TiDB 组件及客户端 TLS 证书配置
+
+## 新增 CRD
+
+- 新增 `TidbMonitor` 用于部署集群监控
+- 新增 `TidbInitializer` 用于初始化集群
+- 新增 `Backup`、`BackupSchedule`、`Restore` 用于备份恢复集群
+- 新增 `TidbClusterAutoScaler` 实现[集群自动伸缩功能](enable-tidb-cluster-auto-scaling.md) (开启 `AutoScaling` 特性开关后使用)
```
Comment (ran-huang) [inline zh/whats-new-in-v1.1.md:23]:
```suggestion
- 新增 `TidbClusterAutoScaler` 实现[集群自动伸缩功能](enable-tidb-cluster-auto-scaling.md)（开启 `AutoScaling` 特性开关后使用）
```

File: zh/whats-new-in-v1.1.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+- 新增可选的[准入控制器](enable-admission-webhook.md)改进升级、扩缩容体验，并提供灰度发布功能
+- 新增可选的[增强型 StatefulSet 控制器](advanced-statefulset.md)，提供对指定 Pod 进行删除的功能（开启 `AdvancedStatefulSet` 特性开关后使用）
+- `tidb-scheduler` 支持任意维度的 HA 调度和 Preemption 支持
+- 备份与恢复支持将 S3 和 GCS 作为远端存储
+- [优雅重启 TiDB 集群组件](restart-a-tidb-cluster.md)
+- tidb-lightning chart [导入集群数据](restore-data-using-tidb-lightning.md)
+- tidb-drainer chart [部署、管理 TiDB Drainer](deploy-tidb-binlog.md#部署-drainer)
+- tikv-importer chart [部署、管理 TiKV Importer](restore-data-using-tidb-lightning.md#部署-tikv-importer)
+
+完成发布日志参见 [1.1 CHANGE LOG](https://github.com/pingcap/tidb-operator/blob/master/CHANGELOG-1.1.md) 。
```
Comment (ran-huang) [inline zh/whats-new-in-v1.1.md:36]:
```suggestion
完整发布日志参见 [1.1 CHANGE LOG](https://github.com/pingcap/tidb-operator/blob/master/CHANGELOG-1.1.md) 。
```

File: zh/whats-new-in-v1.1.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+TiDB Operator 在 Kubernetes 上部署参见[安装文档](#deploy-tidb-operator.md)，CRD 文档参见 [API References](#api-references.md) 。
```
Comment (ran-huang) [inline zh/whats-new-in-v1.1.md:37]:
```suggestion
TiDB Operator 在 Kubernetes 上部署参见[安装文档](deploy-tidb-operator.md)，CRD 文档参见 [API References](https://github.com/pingcap/docs-tidb-operator/blob/master/zh/api-references.md) 。
```

**Comment by cofyc** [comment]
@ran-huang @DanielZhangQD PR is updated. The content is the same as the v1.1.0 announcement email.

File: zh/whats-new-in-v1.1.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
+---
+title: What's New in v1.1
+category: how-to
+---
+
+# What's New in v1.1
+
+TiDB Operator v1.1 在 v1.0 基础上新增 TiDB 4.0 功能特性支持，TiKV 数据加密、TLS 证书配置等。 新增 TiFlash、TiCDC 新组件部署支持，同时在易用性上做了许多改进，提供与 Kubernetes 原生资源一致的用户体验。以下是主要变化：
```
Comment (ran-huang) [inline zh/whats-new-in-v1.1.md:8]:
```suggestion
TiDB Operator v1.1 在 v1.0 基础上新增 TiDB 4.0 功能特性支持，TiKV 数据加密、TLS 证书配置等。新增 TiFlash、TiCDC 新组件部署支持，同时在易用性上做了许多改进，提供与 Kubernetes 原生资源一致的用户体验。以下是主要变化：
```

File: zh/whats-new-in-v1.1.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+## 扩展性
+
+- TidbCluster CR 支持部署管理 PD Discovery 组件，可完全替代 tidb-cluster chart 管理 TiDB 集群
+- TidbCluster CR 新增 Pump、TiFlash、TiCDC、Dashboard 支持
+- 新增可选的[准入控制器](enable-admission-webhook.md)改进升级、扩缩容体验，并提供灰度发布功能
+- tidb-scheduler 支持任意维度的 HA 调度和调度器 preemption
+- tikv-importer chart [部署、管理 TiKV Importer](restore-data-using-tidb-lightning.md#部署-tikv-importer)
```
Comment (ran-huang) [inline zh/whats-new-in-v1.1.md:16]: ”使用“，”新增“ or other verb you see fit.
```suggestion
- 使用 tikv-importer chart [部署、管理 TiKV Importer](restore-data-using-tidb-lightning.md#部署-tikv-importer)
```

File: zh/whats-new-in-v1.1.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+## 易用性
+
+- 新增 TidbMonitor CR 用于部署集群监控
+- 新增 TidbInitializer CR 用于初始化集群
+- 新增 Backup、BackupSchedule、Restore  CR 用于备份恢复集群。备份、恢复支持 S3 和 GCS
```
Comment (ran-huang) [inline zh/whats-new-in-v1.1.md:22]:
```suggestion
- 新增 Backup、BackupSchedule、Restore CR 用于备份恢复集群。备份、恢复支持 S3 和 GCS
```

File: zh/whats-new-in-v1.1.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+- [优雅重启 TiDB 集群组件](restart-a-tidb-cluster.md)
+
+## 安全性
+
+- 支持 TiDB 集群各组件及客户端 TLS 证书配置 
+- 支持 TiKV 数据存储加密
+
+## 实验性特性
+
+- 新增 `TidbClusterAutoScaler` 实现[集群自动伸缩功能](enable-tidb-cluster-auto-scaling.md) （开启 `AutoScaling` 特性开关后使用）
```
Comment (ran-huang) [inline zh/whats-new-in-v1.1.md:32]:
```suggestion
- 新增 `TidbClusterAutoScaler` 实现[集群自动伸缩功能](enable-tidb-cluster-auto-scaling.md)（开启 `AutoScaling` 特性开关后使用）
```

File: zh/whats-new-in-v1.1.md [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+- 新增可选的[增强型 StatefulSet 控制器](advanced-statefulset.md)，提供对指定 Pod 进行删除的功能 （开启 `AdvancedStatefulSet` 特性开关后使用）
```
Comment (ran-huang) [inline zh/whats-new-in-v1.1.md:33]:
```suggestion
- 新增可选的[增强型 StatefulSet 控制器](advanced-statefulset.md)，提供对指定 Pod 进行删除的功能（开启 `AdvancedStatefulSet` 特性开关后使用）
```

**Pull request merged**